ci/test_wheel.sh

```diff
@@ -1,40 +0,0 @@
-#!/bin/bash
```

Comment:
@alexbarghi-nv this looks like it's not used on #53.

I strongly suspect the version here was adopted from https://github.com/rapidsai/cugraph/blob/branch-24.12/ci/test_wheel.sh, which in the `cugraph` repo is never used with `cugraph-dgl` / `cugraph-pyg`: https://github.com/search?q=repo%3Arapidsai%2Fcugraph%20%22test_wheel.sh%22&type=code

I strongly suspect `test_wheel.sh` could just be removed, and that the `ci/wheel_smoke_test_${package_name}.py` scripts being proposed on #53 are unnecessary, and covering functionality already covered by the `ci/test_wheel_${package_name}.sh` scripts.

Reply:
Yeah, this can be taken out

Reply:
Thanks! I've just pushed changes to #53 removing the smoke-testing stuff that was added there.